tests/problems/space/test_alignment_problem.py

```diff
@@ -235,3 +235,23 @@ def test_pass_arguments(self, adata_space_rotate: AnnData, args_to_check: Mappin
         for arg, val in pointcloud_args.items():
             assert hasattr(geom, val)
             assert getattr(geom, val) == args_to_check[arg]
+
+    def test_alignment_order_preservation(self, adata_space_rotate: AnnData):
```

Comment:
Can we have this test for all policies which we allow for the `AlignmentProblem`?

Reply:
ok added star with reference 0 and 1, and set the threshold to 0.96.